client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -114,6 +136,57 @@ export class DraggablePanelComponent implements OnInit, OnChanges {
     }
   }
 
+  addMarker(incident:any) {
+    if (this.marker) {
+      this.marker.remove()
+      this.marker = null
+    }
+
+    if (this.markerAnimation) {
+      clearInterval(this.markerAnimation)
+    }
+
+    const pointerIcon = L.divIcon({
+      iconSize: [20, 20],
+      iconAnchor: [12, 12],
+      popupAnchor: [10, 0],
+      shadowSize: [0, 0],
+      className: 'animated-icon'
+    });
+    this.marker = L.marker([Number(incident.latitude), Number(incident.longitude)],{icon: pointerIcon})
+    this.marker.on('add',function(){
+        const icon: any = document.querySelector('.animated-icon')
+        incident
+        if (incident.fireOfNoteInd) {
```

Comment:
There is a common function in index.ts called **setDisplayColor** that should work for this

Reply:
thanks, I made the change